app-backend/batch/src/main/scala/com/azavea/rf/batch/ingest/model/TargetBand.scala

```diff
@@ -0,0 +1,12 @@
+package com.azavea.rf.batch.ingest.model
```

Comment:
The intention here is to set us up for naming bands, but doesn't alter anything really for existing ingests/data - correct?

Reply:
Yeah -- it exists for the magic json deserialization and so I could keep the format of the incoming data consistent regardless of datasource (see e.g. [geotiff bandmap changes](https://github.com/raster-foundry/raster-foundry/blob/905951322737cad1526309071c1b129c67267a60/app-tasks/rf/src/rf/ingest/geotiff_ingest.py#L47-L48) and [landsat 8 bandmap changes](https://github.com/raster-foundry/raster-foundry/blob/905951322737cad1526309071c1b129c67267a60/app-tasks/rf/src/rf/ingest/landsat8_ingest.py#L54-L58)